server/handlers/custom-route-handler.js

```diff
@@ -13,6 +15,38 @@ exports.customRouteHandler = function customRouteHandler(req, res, next, { confi
       if(page.type === 'redirect') { 
         return res.redirect(page["status-code"], page["destination-path"]);
       }
+
+      if(page.type === 'static-page') {
+        if(page.metadata.header || page.metadata.footer) {
```

Comment:
add `|| page.metadata.layout`.

Reply:
Did we update the data format we receive from the custom-urls api ?

Reply:
Actually we didn't yet so will ask the platform team to update it but for now i think we have to continue with this.